src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/reverse_ref.cl

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/batch_headers/data_types.cl"
+#include "include/batch_headers/fetch_data.cl"
+
+#define BATCH_INDEX 0
+#define FEATURE_INDEX 1
+#if INPUT0_DIMS == 4
+    #define Y_AXE_INDEX 2
```

Comment:
In my opinion "AXE" can be omitted in these identifiers without loss of readability.
Or at least replace it with "AXIS" )

Reply:
done